### ISSUE TITLE
IX-F Importer: Handling of members without ip-address information

### ISSUE THREAD

**Issue by vegu** [peeringdb/peeringdb#849]
Split out from https://github.com/peeringdb/peeringdb/issues/795#issuecomment-678951764

This issue is blocked by #480, which is currently still under discussion.

In the case of #480 being implemented where netixlans can exist without ip-addresses the importer will
need to be adjusted to allow processing of ix-f member entries without ip-addresses as well. This is currently
blocked by validation check of the ix-f data.

Furthermore, at this moment #480 is specified to allow a network to have multiple netixlans without ipaddress
information at the same exchange - e.g., we cannot use the tuple of (asn, ip4(null) ,ip6(null)) to uniquely identify
a netixlan at the exchange and the importer will need to be able to do a stateless sync of those types of entries.

**Comment by arnoldnipper** [comment]
@vegu what is the purpose of this issue? jfyi?

**Comment by vegu** [comment]
#795 was getting chaotic and tracking several different issues, this was one of the issues that we split out from that ticket.

This is very closely tied to 480, but it seemed cleaner to keep it in it's own issue since it deals with the importer's implementation against the changes proposed there. Would you prefer to have it be a part of #480 instead?

**Comment by arnoldnipper** [comment]
If @pc decide in #480 that an asn-only entry is fine then it belongs there, @vegu. Hence my question.

**Comment by vegu** [comment]
That's fine, main goal was to split it out of #795 and have it tracked somewhere. Can close this one and make a note in #480 for it.

**Comment by arnoldnipper** [comment]
#480 has been decided on. A `netixlan` object has to have an ipaddr{4,6}

**Comment by arnoldnipper** [comment]
Is this issue still needed, @vegu?

**Comment by vegu** [comment]
@arnoldnipper Dont think so - as #480 was closed keeping the current state of netixlan ipaddress requirements there is nothing else to do here. Closing this.
